db/sqlite3/sqlite3_test.go

```diff
@@ -0,0 +1,262 @@
+// Copyright (c) 2018 IoTeX
```

Comment:
Also, can we conform the test code with rds?

Reply:
sure

Reply:
BTW, copyright **2019**